python/Ganga/Core/exceptions.py

```diff
@@ -270,6 +270,34 @@ def listFailedJobs(self):
     def getOriginalJobError(self, id):
         return self.details.get(id)
 
+
+class CredentialsError(GangaException):
+    """
+    Base class for credential-related errors
+    """
+    pass
```

Comment:
I don't think the pass is strictly needed here and I have seen several tools autostrip this from codebases.